bbk-requirements.md

```diff
@@ -0,0 +1,76 @@
+
+The payments industry has indicated that SPC would further benefit from a device binding capability. As WebAuthn passkeys can now be synced, it can be argued that they no longer meet strict 2FA requirements (being no longer a signal of device possession), and so SPC (like WebAuthn) is reduced to a single factor (biometric or possession, depending on the authentication method used).
+
+The Web Authentication Working Group has considered but not adopted device binding proposals (cf. [SPK](https://github.com/w3c/webauthn/pull/1957) and [DPK](https://github.com/w3c/webauthn/issues/1658)).
+
+Therefore, the Web Payments Working Group plans to add a "browser-based key (BBK)" SPC. This document endeavors to capture requirements for the BBK functionality.
+
+## Assumptions
+
+* A relying party will perform some ID &amp; V process before trusting a (new) BBK. That ID &amp; V process might take place before a Web Authentication registration (and thus, if the BBK is returned as part of the Web Authentication registration, the RP would not likely step up the user a second time). In the case of a synched passkey, when the RP first sees a BBK on a new user device, in the absence of other trust signals, the RP would likely perform some ID &amp; V process in order to trust the new BBK, and we consider that an acceptable user experience on a new device.
+
+## Requirements
+
+***Notes:***
+
+* These requirements are not prioritized.
+* "MUST", "SHOULD", and "MAY" are used per [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+
+### Association with passkeys
+
+* A given passkey should have only one associated BBK at any given time for a given user agent user profile.
+
+* A BBK must only ever be associated with one passkey.
+
+* If a BBK is deleted, the user agent should generate a new BBK associated with the same passkey.
+
+* At authentication time, the Web Authentication challenge will be signed by the passkey and the associated BBK.
+
+* To link the BBK to the passkey cryptographically, the BBK public key should be added in the passkey client data.
+
+### User agent user profiles
+
+* A user agent may reuse the same BBK across user profiles of the same user agent instance.
+
+### Device binding
+
+* To meet anticipated security requirements, issuance of the BBK by the user agent should involve a device-binding process that ensures a unique connection between the user agent (user profile) and the device. This may be, for example, through hardware crypto-security (e.g., TPM), keys stored in the secure element, or registration of the web browser linking a browser to a device.
+
+* Once a BBK has been bound to a device, it must only ever be bound to that device.
+
+* For environments where a device-binding process is not readily available, the BBK may still provide value. Not every transaction requires the same level of security (e.g., low-value transactions), and so even a BBK that is not device-bound may be useful.
```

Comment:
Is the usage of the word "may" here in alignment with "MAY" from RFC 2219?

We're striving to make BBK always device-bound. We achieve that by using OS-level APIs to access the TPM.

Reply:
I will change this to say: "The user agent may return a BBK even in environments where a device-binding process is not readily available." I don't yet know whether it should be a "SHOULD."